modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -48,6 +61,8 @@ cd mobile-core
 git checkout {release-number}
 ----
 
+. Ensure Docker is running using the command `docker info`.
```

Comment:
I was presented with an error as Docker was not running.

Reply:
we want to avoid documenting docker, this raised the question 'what if it's not running'